assets/large_language_models/prompts/one_shot_financial_news_summarization/prompt/prompt.yaml

```diff
@@ -0,0 +1,37 @@
+$schema: https://azuremlschemas.azureedge.net/latest/prompt.schema.json
+
+_type: chat
+
+messages:
+- role: system
+  content: |
+    As an AI assistant with extensive experience in identifying insights from financial data, such as trading data, commodity data, and exchange data, you focus on carefully examining the information provided within the prompt. You are committed to maintaining accuracy and never fabricating information. If uncertain about any aspect, you will explicitly state your uncertainty in the response.
+    Your expertise is tailored to serve various financial industry professionals, including Financial Analysts, Investment Bankers, and others. All generated content is specifically crafted to cater to the needs of these industry members.
+    When presenting calculated figures, it is crucial to provide a clear explanation of the derivation process, ensuring users fully comprehend how the numbers were obtained. You are meticulous in maintaining the accuracy of all mathematical calculations and never resort to using fabricated numbers. Please summarize the following news article for financial analysts in a concise manner. Your summary should only include the most important information and avoid unnecessary words.
+- role: user
+  content: "Please summarize the following news article for financial analysts in a concise manner. Your summary should only include the most important information and avoid unnecessary words.\nnews article: Artificial Intelligence (AI) tech is an $800 billion market opportunity over the next decade as it pervades into the enterprise and consumer tech space, said Wedbush analyst Daniel Ives in a recent note.  Despite's Apple, Inc.\xE2\u20AC\u02DCs not-so-proactive approach, the analyst said Cupertino could be the \"one to benefit no matter where the industry goes.\"  Apple Preps To Press The Pedal? Given Apple is a leading player in nearly everything regarding tech and consumer technology, the company will likely step up its AI efforts over the coming months, Ives said in a recent note.  The company's thrust on AI is evident from the 2020 acquisition of Xnor.ai, a spin-off from the non-profit Allen Institute for AI, he said. The tech giant would have the opportunity to leverage Xnor.ai capabilities\
```

Comment:
can you do the | approach for here too?

Reply:
and can we use this style for all your multiline string?

Reply:
fixed
The issue was related to special characters in the text that yaml package could not process right.